lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -402,7 +409,8 @@ define((require, exports, module) ->
 
     equationRangeHandler = new EquationRangeHandler(editor)
 
-    contextHandler = new ContextHandler(editor, jqEditorContainer, popoverHandler, equationRangeHandler, I18N)
+    contextHandler = new ContextHandler(editor, popoverHandler, equationRangeHandler, I18N)
+    exports.reset = -> if contextHandler.contextPreviewExists then contextHandler.destroyEverything()
```

Comment:
Also, pls refactor exports of reset and SelectionHandler. Now they will be exported only after the call to `setupPreviewer`. I suggest either exposing them always (probably returning null or doing nothing if setup has not been called) or returning an object with these functions from `setupPreviewer` so that the caller had a chance to call `reset` only when he previously called `setup`